principles/fp-001-open.md

```diff
@@ -18,7 +18,7 @@ OBO Foundry ontologies are resources for the entire biological and biomedical co
 
 ### For ontology providers
 
-OBO Foundry Ontologies MUST EITHER be released under a [Creative Commons](https://creativecommons.org/) [CC-BY license version 3.0](https://creativecommons.org/licenses/by/3.0/) or later, OR released into the public domain under [CC0](https://creativecommons.org/publicdomain/zero/1.0/). It should be clearly stated in the ontology file.
+OBO Foundry Ontologies MUST EITHER be released under a [Creative Common Attribution 3.0 International (CC BY 3.0) license or later](https://creativecommons.org/licenses/by/3.0/) or later, OR released into the public domain under [a Creative Commons CC0 public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/). The license MUST be [clearly stated](https://github.com/INCATools/ontology-development-kit/blob/master/docs/License.md) using the http://purl.org/dc/terms/license property in the ontology file.
```

Comment:
Still possibility for confusion - what comes after http://purl.org/dc/terms/license? Maybe someone will put free text instead of the proper software URL. Could you give an example, plus a list to the controlled vocabulary that could be used?

Reply:
Thank you @cthoyt, done.. Better now?